packages/terra-application-menu-layout/tests/nightwatch/application-menu-spec.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-unused-expressions */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { resizeTo } = require('terra-toolkit/lib/nightwatch/responsive-helpers');
+
+module.exports = resizeTo(['large'], {
```

Comment:
With this view change at different viewports or is that the layout component that makes these adjustments?

Reply:
It's up to the layout and individual items to respond. These are now just husks.